### ISSUE TITLE
Document requirements

### ISSUE THREAD

**Issue by lweeks** [zhang-bob/Fibonacci#1]
Hi Bob,

Can you add a requirements.txt file to this with what Python packages are required, e.g. a `pip freeze` result, as well as some documentation on the environment required to get this going, such as Django. Thanks.

**Comment by zhang-bob** [comment]
yes, you are right. today I add requirments.txt  and also update README, to add more details on how to make django ready and how to map our project fibonacci with this framework
